### PR TITLE
updated benchmarks for hypeshift

### LINKED ISSUE #159: Hosted cluster benchmarks

### PR DESCRIPTION
**Pull request by mukrishn**
### Description
Updated benchmarks for hypershift to adopt [new](https://github.com/cloud-bulldozer/e2e-benchmarking/pull/335) templates

[kube-burner report
](http://grafana.apps.observability.perfscale.devcluster.openshift.com/d/n8_lX3-nk/kube-burner-report-v2?orgId=1&from=1649169444793&to=1649170072278&var-Datasource=Observability%20-%20ripsaw-kube-burner&var-sdn=openshift-sdn&var-sdn=openshift-ovn-kubernetes&var-job=baseline-performance-workload&var-job=node-density&var-job=node-density-heavy&var-uuid=48581a0f-hosted-2-node-density-20220405&var-master=&var-worker=ip-10-0-129-121.us-west-2.compute.internal&var-worker=ip-10-0-129-209.us-west-2.compute.internal&var-infra=&var-namespace=All&var-latencyPercentile=P99)

### CODE REVIEW

File: scripts/playground/cleanup.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@ set -a
 usage() { echo "Usage: $0 [-p <string> (airflow password)]" 1>&2; exit 1; }
 GIT_ROOT=$(git rev-parse --show-toplevel)
 source $GIT_ROOT/scripts/common.sh
-_airflow_namespace=$_remote_user-$_branch-airflow
+_airflow_namespace=$_remote_user-$_branch_display_name-airflow
```
Comment (Harshith-umesh) [inline scripts/playground/cleanup.sh:6]: why change this?

File: dags/openshift_nightlies/scripts/install/hypershift.sh [hunk 2]
```diff
@@ -89,7 +89,7 @@ create_cluster(){
     export REPLICA_TYPE=$(cat ${json_file} | jq -r .hosted_control_plane_availability)   
     BASEDOMAIN=$(_get_base_domain $(_get_cluster_id ${MGMT_CLUSTER_NAME}))
     echo $PULL_SECRET > pull-secret
-    hypershift create cluster aws --name $HOSTED_CLUSTER_NAME --node-pool-replicas=$COMPUTE_WORKERS_NUMBER --base-domain $BASEDOMAIN --pull-secret pull-secret --aws-creds aws_credentials --region $AWS_REGION --control-plane-availability-policy $REPLICA_TYPE --network-type $NETWORK_TYPE --instance-type $COMPUTE_WORKERS_TYPE
+    hypershift create cluster aws --name $HOSTED_CLUSTER_NAME --node-pool-replicas=$COMPUTE_WORKERS_NUMBER --base-domain $BASEDOMAIN --pull-secret pull-secret --aws-creds aws_credentials --region $AWS_REGION --control-plane-availability-policy $REPLICA_TYPE --network-type $NETWORK_TYPE --instance-type $COMPUTE_WORKERS_TYPE # --control-plane-operator-image=quay.io/hypershift/hypershift:latest
```
Comment (dry923) [inline dags/openshift_nightlies/scripts/install/hypershift.sh:92]: hanging comment at the end of line

File: dags/openshift_nightlies/config/benchmarks/hosted-control-plane.json [hunk 3]
```diff
@@ -16,7 +21,9 @@
                 "JOB_TIMEOUT": "18000",
                 "QPS": "20",
                 "BURST": "20",
-                "STEP_SIZE": "30s",
+                "STEP_SIZE": "60s",
+                "HYPERSHIFT": "true",
```
Comment (Harshith-umesh) [inline dags/openshift_nightlies/config/benchmarks/hosted-control-plane.json:25]: where is this env var being used?

Comment (mukrishn) [reply dags/openshift_nightlies/config/benchmarks/hosted-control-plane.json:25]: @Harshith-umesh `HYPERSHIFT` is used in e2e script [here](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/common.sh#L40) for conditional tasks and `STEP_SIZE` is used in kube-burner ripsaw yaml [here](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/kube-burner-crd.yaml#L45) for prometheus step size.

Comment (Harshith-umesh) [reply dags/openshift_nightlies/config/benchmarks/hosted-control-plane.json:25]: interesting.

File: dags/openshift_nightlies/tasks/benchmarks/e2e.py [hunk 4]
```diff
@@ -76,6 +76,14 @@ def __init__(self, dag, config: DagConfig, release: OpenshiftRelease, task_group
                 "AWS_DEFAULT_REGION": self.aws_creds['aws_region_for_openshift']
             }
     
+        if self.release.platform == "hypershift":
+            mgmt_cluster_name = release._generate_cluster_name()
+            self.env = {
+                "THANOS_RECEIVER_URL": var_loader.get_secret("thanos_receiver_url"),
+                "PROM_URL": var_loader.get_secret("thanos_querier_url"),
+                "MGMT_CLUSTER_NAME": f"{mgmt_cluster_name}.*",
```
Comment (Harshith-umesh) [inline dags/openshift_nightlies/tasks/benchmarks/e2e.py:84]: why the * ?

Comment (mukrishn) [reply dags/openshift_nightlies/tasks/benchmarks/e2e.py:84]: Used in a wildcard query [here](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/metrics-profiles/hypershift-metrics.yaml#L3)

**Review by Harshith-umesh — APPROVED** [review]
LGTM

**Pull request merged**